Tests/Controller/FakeControllerTest.php

```diff
@@ -71,4 +71,14 @@ public function testReadAndSaveAction()
 
         $this->assertFileExists($content, sprintf('file %s should exist', $content));
     }
+
+    /**
+     * @inheritdoc
+     */
+    protected static function getKernelClass()
+    {
+        require_once __DIR__.'/../app/AppKernel.php';
```

Comment:
even better: configure the `autoload_dev` in composer.json to be able to autoload this class (I think it is even already able to do it), making this requiring useless